src/utils/reducer.js

```diff
@@ -1,11 +1,11 @@
 import * as actionTypes from '../constants/actionTypes'
 
 export const initialState = {
```

Comment:
after looking over this reducer, I think it should be replaced with some `useState` hooks.  There isn't any complex side effects from setting data, it looks like simple persistence.